src/kOS/Communication/ConnectivityManager.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace kOS.Communication
+{
+    public interface ConnectivityManager
+    {
+        double GetDelay(Vessel vessel1, Vessel vessel2);
+    }
+}
```

Comment:
What are you seeing as the meaning of "inter-processor" in command-line mode, given that the command-line mode just has the one kOS CPU?